go.mod

```diff
@@ -12,16 +12,13 @@ require (
 	github.com/gobuffalo/events v1.4.2
 	github.com/gobuffalo/flect v0.2.5
 	github.com/gobuffalo/genny/v2 v2.0.9
-	github.com/gobuffalo/here v0.6.6 // indirect
```

Comment:
Oh, it reverted my change merged yesterday.

If go.mod has a line with indirect comment, usually it could mean intentionally added/fixed version.